Spec/RealtimeClientConnection.swift

```diff
@@ -1660,6 +1660,34 @@ class RealtimeClientConnection: QuickSpec {
 
                 }
 
+                // RTN15f
+                it("ACK and NACK responses for published messages can only ever be received on the transport connection on which those messages were sent") {
+                    let options = AblyTests.commonAppSetup()
+                    options.disconnectedRetryTimeout = 0.5
+                    let client = AblyTests.newRealtime(options)
+                    defer { client.close() }
+                    let channel = client.channels.get("test")
+                    var resumed = false
+                    waitUntil(timeout: testTimeout) { done in
+                        client.connection.once(.Connected) { _ in
+                            channel.publish(nil, data: "message") { _ in
```

Comment:
Is called twice? Can you clarify? The `publish` only receives the `ACK` (callback is called) after the connection has been resumed.